WatchParty/backend/models/follow.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, DataTypes) => {
+  const Follow = sequelize.define("Follow", {
+    followee: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      validate: { notNull: true }
+    },
+    follower: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      validate: { notNull: true }
+    }
+},{
+    freezeTableName: true
+	});
+	Follow.associate = function(models) {
```

Comment:
These two are belongs-to-many (you can specify how  here https://sequelize.readthedocs.io/en/latest/docs/associations/#belongs-to-many-associations )

Reply:
Changed both to belongs-to-many.